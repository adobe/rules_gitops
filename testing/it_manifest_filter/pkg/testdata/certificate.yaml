apiVersion: certmanager.k8s.io/v1alpha1
kind: Certificate
metadata:
  name: actv-notification-certificate
  namespace: namespace
spec:
  secretName: actv-notification-tls
  issuerRef:
    name: letsencrypt-prod
    kind: ClusterIssuer
  commonName: actv-notification-service.@k8s.namespace@.@k8s.cluster@.k8s.tubemogul.info
  dnsNames:
    - actv-notification-service.@k8s.namespace@.@k8s.cluster@.k8s.tubemogul.info
  acme:
    config:
      - dns01:
          provider: route53
        domains:
          - actv-notification-service.@k8s.namespace@.@k8s.cluster@.k8s.tubemogul.info
