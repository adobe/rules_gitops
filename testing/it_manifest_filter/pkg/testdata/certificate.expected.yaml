apiVersion: certmanager.k8s.io/v1alpha1
kind: Certificate
metadata:
  name: actv-notification-certificate
  namespace: namespace
spec:
  acme:
    config:
    - dns01:
        provider: route53
      domains:
      - actv-notification-service.@k8s.namespace@.@k8s.cluster@.k8s.tubemogul.info
  commonName: actv-notification-service.@k8s.namespace@.@k8s.cluster@.k8s.tubemogul.info
  dnsNames:
  - actv-notification-service.@k8s.namespace@.@k8s.cluster@.k8s.tubemogul.info
  issuerRef:
    kind: ClusterIssuer
    name: letsencrypt-staging
  secretName: actv-notification-tls
